DepositeCalcTests/Pages/RemindPasswordView.cs

```diff
@@ -0,0 +1,81 @@
+﻿using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using System;
+using SeleniumExtras.WaitHelpers;
+using System.Linq;
+
+namespace DepositeCalcTests.Pages
+{
+    internal class RemindPasswordView
+    {
+        protected readonly IWebDriver driver;
+        public RemindPasswordView(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
```

Comment:
This view is a part of 'Login' page, it cannot contain 'RemindPassword' button.
```suggestion
        private readonly IWebDriver _driver;
        private readonly IWebElement _remindPasswordBtn;
        public RemindPasswordView(IWebDriver driver, IWebElement button)
        {
            _driver = driver;
            _remindPasswordBtn = button;
        }
```